ipa-core/src/report.rs

```diff
@@ -418,19 +420,11 @@ where
     TV: WeakSharedValue,
     TS: WeakSharedValue,
 {
-    pub timestamp: Replicated<TS>,
-    pub mk_oprf: u64,
-    pub event_type: EventType,
+    pub match_key: Replicated<BA64>,
```

Comment:
I will need to revisit these types to reflect correct input but wil handle in next PR.
I think we need 
match_key: Replicated<BA64>,
is_trigger: Boolean,
breakdown_key: Replicated<BK>,
trigger_value: Replicated<TV>
timestamp: Replicated<TS>

or should bk and ts be sent in clear?